client/components/place/place-list/place-list.template.html

```diff
@@ -213,21 +213,26 @@
     </div>
 
     <div class="row">
+      <!-- Spinner -->
+      <div id="spinner" class="spinner">
+        <div class="dot1"></div>
+        <div class="dot2"></div>
+      </div>
       <!-- Leaflet map -->
       <div class="map-wrapper">
         <div id="map"></div>
       </div>
       <!-- Div for Popular places -->
-      <div class="popular-places-wrapper" ng-hide="$ctrl.hidePopularPlaces">
-        <a href="" ng-click="$ctrl.checkPopularPlaces()">
+      <div class="popular-places-wrapper ng-hide" ng-hide="$ctrl.hidePopularPlaces">
+        <a href="" class="close-button" ng-click="$ctrl.checkPopularPlaces()">
           <i class="fa fa-times" aria-hidden="true"></i>
         </a>
         <div class="popular-div-style">
         </div>
       </div>
       <!-- Div for Popular tracks -->
-      <div class="popular-tracks-wrapper" ng-hide="$ctrl.hidePopularTracks">
-        <a href="" ng-click="$ctrl.checkPopularTracks()">
+      <div class="popular-tracks-wrapper ng-hide" ng-hide="$ctrl.hidePopularTracks">
```

Comment:
Maybe refactor this to `ng-show` and `$ctrl.showPopularTracks`?
Because if it's **hidden by default** you need to **show** this.